requirements.txt

```
@@ -0,0 +1,21 @@
+pillow
+pandas~=2.2.3
+tqdm~=4.67.1
+requests~=2.32.3
+opencv-python~=4.11.0.86
+scikit-learn~=1.6.1
+faiss-cpu~=1.10.0
+flask~=3.1.0
+matplotlib~=3.10.1
+scipy~=1.15.2
+scikit-image~=0.25.2
+tensorflow-cpu
+umap-learn~=0.5.7
+keras~=3.9.0
+accelerate>=0.26.0
+transformers~=4.49.0
+peft==0.14.0
+numpy~=2.1.3
+werkzeug~=3.1.3
+torch~=2.6.0
```

Can we use `2.7.1` ? 
```suggestion
torch~=2.7.1
```